### ISSUE TITLE
Is it possible to use this library with a single monocular RGB camera (no depth)

### ISSUE THREAD

**Issue by slooi** [UZ-SLAMLab/ORB_SLAM3#776]
I'm trying to localize an agent within an environment using only a single monocular RGB camera. Would it still be possible to use this library for that purpose? Or would I have to use a library like https://github.com/raulmur/ORB_SLAM which I believe supports this. Additionally, if anyone knows of other libraries or technologies which could help with this task please let me know.

**Comment by KamiliArsyad** [comment]
Yes. You can just set the sensor setting to monocular when you initialize the system.
